soc/espressif/common/loader.c

```diff
@@ -68,28 +72,32 @@
 
 #define HDR_ATTR __attribute__((section(".entry_addr"))) __attribute__((used))
 
+#if !defined(CONFIG_SOC_ESP32_APPCPU) && !defined(CONFIG_SOC_ESP32S3_APPCPU)
+#define PART_OFFSET FIXED_PARTITION_OFFSET(slot0_partition)
```

Comment:
Does it make sense to use `APPCPU_SLOT_OFFSET` instead of `PART_OFFSET`?

Reply:
no it doesn't, the code is used in both cases PRO_CPU and APP_CPU